swap_meet/vendor.py

```diff
@@ -1,2 +1,136 @@
+from .item import Item
+
+# WAVE 1
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, new_item):
+        self.inventory.append(new_item)
+        return new_item
+
+    def remove(self, item_to_remove):
+        if item_to_remove in self.inventory:
+            self.inventory.remove(item_to_remove)
+            return item_to_remove
+        return False
```

Comment:
Prefer to use a [guard clause](https://refactoring.guru/replace-nested-conditional-with-guard-clauses) and invert the logic here. 

We should check for invalid input and immediately return `False`. If the input is valid then the main logic on lines 16-17 can be unindented. We should seek to keep our logic as unindented as possible, you can read more about this concept [here](https://nelkodev.com/en/blog/the-art-of-indentation-key-to-readable-and-maintainable-code/).

```suggestion
        if item_to_remove not in self.inventory:
            return False
        # your logic to remove the item outside a conditional block
```

Reply:
Refactored remove method to use a guard clause, handling invalid input early and reducing indentation for the main logic.